pmaudit

```diff
@@ -490,6 +490,17 @@ def main():
         '-d', '--depsfile',
         metavar='FILE',
         help="save audit data in makefile format to FILE")
+    parser.add_argument(
+        '--shell',
+        metavar='SHELL_COMMAND',
+        help="Name of shell to run (default=SHELL else /bin/sh)")
+    parser.add_argument(
+        '--shellflags',
+        metavar='SHELL_FLAGS',
```

Comment:
As before, I see no value in using metavar to add an underscore. If you like the underscore, name the option --shell-flags and it will automatically be referred to as SHELL_FLAGS. Personally I don't like the underscore but don't care very much.

Reply:
Metavariable "SHELL_FLAGS" removed.